generator/src/test/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGeneratorTest.java

```diff
@@ -110,4 +101,30 @@ public void testGeneratesValidExpressions() throws Exception {
     }
   }
 
+  private List<Stmt> makeVariableListAndAssignMutatedExpression(BasicType basicType,
+                                                                ShadingLanguageVersion shadingLanguageVersion,
+                                                                int variableNumber) {
```

Comment:
In light of my overall suggested change, can you call the new parameter "numberOfAssignments"?

And can you name the method "makeMutatedExpressionAssignments"?  (Avoid "List" in the name, as the return type documents this.)

Reply:
Done!